.nvmrc

```diff
@@ -1 +1 @@
-v18.15.0
\ No newline at end of file
+v20.12.2
```

Comment:
I think we should update the Node.js version mentioned in the documentation to >= 21.0.0, because in the package.json file, we have also defined the engines node as >= 21.0.0